src/lib/dev_util/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { v4 as uuidv4, parse as uuidParse } from "uuid";
+import { bytesToBigInt } from "babyjubjub-ecdsa";
+import { cardPubKeys } from "../dev_mockData/cardPubKeys";
+
+export const getJubmojiPubKeyFromIndex = (index: number): string => {
+  return cardPubKeys[index].pubKeyWeierstrass;
+};
+
+export const getRandomNullifierRandomness = (): bigint => {
+  const uuid = uuidv4();
```

Comment:
I've heard UUIDs aren't good sources of randomness actually, cuz of some entropy stuff. Should probably use a dedicated RNG (`crypto.getRandomValues` comes to mind)

Reply:
Wow didn't know about browser crypto...